tools/partyplanner

```diff
@@ -12,7 +12,8 @@ BOTTLEROCKET_ROOT_TYPECODE="5526016a-1a97-4ea4-b39a-b7c8c6ca4502"
 BOTTLEROCKET_HASH_TYPECODE="598f10af-c955-4456-6a99-7720068a6cea"
 BOTTLEROCKET_RESERVED_TYPECODE="0c5d99a5-d331-4147-baef-08e2b855bdc9"
 BOTTLEROCKET_PRIVATE_TYPECODE="440408bb-eb0b-4328-a6e5-a29038fad706"
-BOTTLEROCKET_DATA_TYPECODE="626f7474-6c65-6474-6861-726d61726b73"
+BOTTLEROCKET_DATA_A_TYPECODE="626f7474-6c65-6474-6861-726d61726b73"
+BOTTLEROCKET_DATA_B_TYPECODE="74746f42-656c-6b72-7444-617461416c74"
```

Comment:
We may want to keep the same typecode for both partitions, otherwise the behavior of `repart-local.service` will be inconsistent:
* if DATA-A was picked for BOTTLEROCKET-DATA, then the partition will be resized on each subsequent and the filesystem extended
* if DATA-B was picked for BOTTLEROCKET-DATA, then the partition won't be resized and the filesystem won't grow

I think the new implementation would be fine with reusing the type code.

Reply:
I missed your question from yesterday but I think my suggestion here might answer it, if using the same type code is OK now.